index.html

```diff
@@ -3,175 +3,176 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Swift DevRoom @ FOSDEM 2025</title>
+    <title>Swift pre-FOSDEM Community Event 2026</title>
     <link rel="stylesheet" href="style.css"/>
 </head>
 <body class="gradient-background">
 <div class="main-container">
     <div class="centered-container">
         <img src="images/swift-logo.svg" alt="Swift programming language logo" class="logo">
-        <h1>Swift DevRoom at FOSDEM 2025</h1>
-        <p>The Swift open source community will be participating in this year's FOSDEM with a <a
-                href="https://fosdem.org/2025/news/2024-10-28-devrooms-announced/" target="_blank">DevRoom</a>.</p>
-        <p>FOSDEM 2025 takes place in Brussels, Belgium on <b>February 1st-2nd, 2025</b>.</p>
-        <p>Join us to discuss, share, and build with <a href="https://swift.org" target="_blank">Swift</a> at FOSDEM.
+        <h1>Swift pre-FOSDEM Community Event 2026</h1>
+        <p>The Swift open source community will be participating in this year's FOSDEM in the form of a community event the day before the conference, along with talks in multiple tracks.</p>
+        <p>
+            Brussels, Belgium on Friday, <b>January 30th, 2026</b>.<br/>
+            FOSDEM 2026 takes place on January 31st - February 2nd, 2026.
         </p>
-        <p>This page contains information about the Call For Proposals (CFP) process and other information you
-            might find useful for planning participation in the Swift DevRoom.</p>
+        <p>Join us to discuss, share, and build with <a href="https://swift.org" target="_blank">Swift</a>!</p>
     </div>
+    
+
     <section class="left-container">
-        <h2>What is FOSDEM?</h2>
-        <p>FOSDEM is a free event for software developers to meet, share ideas, and collaborate.</p>
+        <h2>Call for Proposals</h2>
```

Comment:
Can we move the whole CFP portion after line 69 AKA after the fringe event has been introduced/contextualized? Will provide for a better flow.